pkg/corerp/frontend/controller/applications/testdata/graph-app-directroute-out.json

```diff
@@ -0,0 +1,23 @@
+[
+    {
+        "connections": [
+            {
+                "direction": "Inbound",
+                "id": "/planes/radius/local/resourcegroups/default/providers/Applications.Core/containers/backendapp"
+            }
+        ],
+        "id": "/planes/radius/local/resourcegroups/default/providers/Applications.Core/containers/frontend",
+        "name": "frontend",
+        "outputResources": [],
+        "provisioningState": "Succeeded",
+        "type": "Applications.Core/containers"
+    },
+    {
+        "connections": [],
```

Comment:
@nithyatsu should the connection also appear here?

Reply:
IICR the data structure annotates the connection in both directions so it's easy to display.

Here's the CLI code that displays it: https://github.com/radius-project/radius/blob/main/pkg/cli/cmd/app/connections/display.go

Reply:
I found that when I fix Inbound direction bug, the connection is generated. So this will be resolved in the following PR.

Reply:
I'm ok with that as long as it's going to be fixed soon.